src/components/media-player/media-duration-cache.js

```diff
@@ -5,6 +5,8 @@ import { getDisposalInfo, DISPOSAL_PRIORITY } from '../../common/di.js';
 
 const logger = new Logger(import.meta.url);
 
+const FAILED_ENTRIES_LIFETIME = 5000;
```

Comment:
In milliseconds, I guess. Maybe good to specify in name (`_IN_MS`) 🤔

Reply:
Sure, renamed.